packages/sui-pde/src/adapters/optimizely/index.js

```diff
@@ -103,7 +102,7 @@ export default class OptimizelyAdapter {
    * @returns {string=} variation name
    */
   activateExperiment({name, attributes}) {
-    if (!this._hasUser()) return null
```

Comment:
the userId check wasn't really needed as the optimizely sdk will check for it too